Documentation/contributing/testing/bpf.rst

```diff
@@ -27,18 +27,11 @@ To run the tests in your local environment, execute the following command from t
 
 .. code-block:: shell-session
 
-    $ make -C test run_bpf_tests
-
-The output is verbose by default. Verbose mode can be disabled by setting the ``V`` option to ``0``:
-
-.. code-block:: shell-session
-
-    $ make -C test run_bpf_tests V=0
+    $ make run_bpf_tests
```

Comment:
I considered this too, but everything beyond the main Makefile is kind of an implementation detail to me. My reasoning is, the docs should point to the entrypoint, we try to keep that stable, and from there on out, the user can figure out how to learn more.

I only found out about this doc snippet by accident because I happened to grep for `bpf_tests`, the old subdir of the Go test runner. If I hadn't, the docs would now be stale, and I'm pretty sure no one would've caught this during review. These details tend to change, and it's not always worth documenting them as long as we leave some pointers to them.